WalletWasabi.Backend/Controllers/BatchController.cs

```diff
@@ -76,7 +76,7 @@ public async Task<IActionResult> GetSynchronizeAsync([FromQuery, Required]string
 				response.Filters = filters;
 			}
 
-			response.CcjRoundStates = ChaumianCoinJoinController.GetStatesCollection();
```

Comment:
Don't rename it. It's not a backwards compatible change. It'd make every client that doesn't upgrade to not be able to coinjoin.

https://wasabiwallet.io/api/v3/btc/Batch/synchronize?bestKnownBlockHash=0000000000000000000e1d4ce08cb39bc27f705ea89347c8c8b4694cdcdf20bb&maxNumberOfFilters=1&estimateSmartFeeMode=CONSERVATIVE